libs/payments/ui/src/lib/server/purchase-details.tsx

```diff
@@ -0,0 +1,209 @@
+
+export type PurchaseDetailsProps = {
+  interval: string;
+  invoice: Invoice;
+  purchaseDetails: {
+    details: string[];
+    subtitle: string;
+    productName: string;
+  };
+};
+
+export async function PurchaseDetails(props: PurchaseDetailsProps) {
+  const { purchaseDetails, invoice, interval } = props;
+  const { currency, listAmount, discountAmount, totalAmount, taxAmounts } =
+    invoice;
+  const { details, subtitle, productName } = purchaseDetails;
+  const exclusiveTaxRates = taxAmounts.filter(
+    (taxAmount) => !taxAmount.inclusive
+  );
+
+  // TODO - Temporary
+  // Identify an approach to ensure we don't have to perform this logic
+  // in every component/page that requires localization.
+  const languages = headers()
+    .get('Accept-Language')
+    ?.split(',')
+    .map((language) => language.split(';')[0]);
+
+  // TODO
+  // Move to instantiation on start up. Ideally getBundle's, generateBundle, is only called once at startup,
+  // and then that instance is used for all requests.
+  // Approach 1 (Experimental): https://nextjs.org/docs/app/building-your-application/optimizing/instrumentation
+  // Approach 2 (Node global): https://github.com/vercel/next.js/blob/canary/examples/with-knex/knex/index.js#L13
+  const l10n = await getBundle(languages);
+
+  return (
+    <div
+      className={`component-card px-4 rounded-t-none tablet:rounded-t-lg`}
+      data-testid="plan-details-component"
+    >
+      <div className="plan-details-header row-divider-grey-200">
+        <div className="plan-details-logo-wrap" style={{}}>
+          <Image
+            // src={webIcon || DEMO_LOGO} // https://nextjs.org/docs/pages/api-reference/components/image#remotepatterns
+            src={DEMO_LOGO}
+            alt={productName}
+            data-testid="product-logo"
+            className="plan-details-icon"
+            width={64}
+            height={64}
+          />
+        </div>
+
+        <div className="text-start text-sm">
+          <h2 id="plan-details-product" className="plan-details-product">
+            {productName}
+          </h2>
+
+          <p className="plan-details-description">
+            {getFormattedMsg(
+              l10n,
+              `plan-price-interval-${interval}`,
+              formatPlanPricing(listAmount, currency, interval),
+              {
+                amount: getLocalizedCurrency(listAmount, currency),
+              }
+            )}
+            &nbsp;&bull;&nbsp;
+            <span>{subtitle}</span>
+          </p>
+        </div>
+      </div>
+
+      <div data-testid="list">
+        <h3 className="text-sm text-grey-600 font-semibold my-4">
+          {l10n.getMessage('plan-details-header')?.value?.toString() ||
+            `Plan Details`}
+        </h3>
+
+        <ul className="row-divider-grey-200 text-grey-400 m-0 px-3 text-sm">
+          {details.map((detail, idx) => (
+            <li className="mb-4 leading-5 marker:text-xs" key={idx}>
```

Comment:
The bullets/markers do not appear next to the details.

Reply:
Oh wow, yup missed this. Fixed, thank you.